File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/18/18.py
# Q18: Print increasing alphabet per row (A, AB, ABC, ABCD, ABCDE).
# Input: An integer n
# Output: Increasing alphabet pattern

n = int(input())
for i in range(n):
    for j in range(i + 1):
        print(chr(65 + j), end="")
    print()

File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/18/18.go
// Q18: Print increasing alphabet per row (A, AB, ABC, ABCD, ABCDE).
// Input: An integer n
// Output: Increasing alphabet pattern

package main

import "fmt"

func main() {
    var n int
    fmt.Scan(&n)
    for i := 0; i < n; i++ {
        row := ""
        for j := 0; j <= i; j++ {
            row += string(rune('A' + j))
        }
        fmt.Println(row)
    }
}
